test/unit/containers/App.spec.js

```diff
@@ -1,33 +1,25 @@
-import React, { View } from "react";
-import { shallow } from "enzyme";
-import MapView from "../../../src/components/MapView.android.js";
-import { expect } from "chai"
-import { App } from "../../../src/containers/App"
-
-describe("App", () => {
+import chai from 'chai';
+chai.should();
 
-  it("should have user's current location", () => {
-    let app = shallow(<App/>);
-    var userLocationView = app.find("View").at(1);
-    expect(userLocationView.find("Text").at(0).children().node).to.contain("LATITUDE: 0");
-    expect(userLocationView.find("Text").at(1).children().node).to.contain("LONGITUDE: 0");
-    expect(userLocationView.find("Text").at(2).children().node).to.contain("LAST UPDATED TIME: 0");
-  });
+import React, { View } from "react-native";
+import { createStore } from 'redux';
+import { shallow } from "enzyme";
+import reducer from '../../../src/redux/reducer';
 
-  describe("MapView", () => {
-    let app = shallow(<App/>);
+import Root from "../../../src/components/Root";
+import App from '../../../src/containers/App';
 
-    it("should exist", () => {
-      expect(app.find(MapView).is("OsmDroidMapView")).to.equal(true);
+describe('App container', () => {
```

Comment:
@brmenchl : this is what i wound up going with to test the plumbing. since plumbing is pretty much now the `App` container's only job, it seemed appropriate to make it a unit test on that class, rather than an integration test. Happy to hear otherwise and change if you think another approach is better!